Mathlib/Data/NNRat/Defs.lean

```diff
@@ -56,7 +56,8 @@ theorem ext : (p : ℚ) = (q : ℚ) → p = q :=
 protected theorem coe_injective : Injective ((↑) : ℚ≥0 → ℚ) :=
   Subtype.coe_injective
 
-@[simp, norm_cast]
+-- We want to use this lemma earlier than the lemma simp can prove it with
```

Comment:
Let's name the lemma in question here: which is the specific one that you want to use it earlier than?